src/odemis/driver/autoscript_client.py

```diff
@@ -208,6 +208,14 @@ def get_hardware_version(self) -> str:
             self.server._pyroClaimOwnership()
             return self.server.get_hardware_version()
 
+    def list_available_channels(self) -> List[str]:
```

Comment:
I think for the xt_client we left this out of the client side, since there was not really a use case for it. I don't see you using it here, is it necessary to have it?